content/webapp/services/prismic/transformers/exhibition-guides.ts

```diff
@@ -77,38 +87,35 @@ export function transformExhibitionGuideToExhibitionGuideBasic(
   }))(exhibitionGuide);
 }
 
-function transformRelatedExhibition(exhibition): Exhibit {
-  return {
-    exhibitType: 'exhibitions',
-    item: undefined,
-    id: exhibition.id,
-    title: (exhibition.data && asText(exhibition.data.title)) || '',
-    description:
-      exhibition.data &&
-      asText(exhibition.data.promo[0].primary.caption[0].text),
-  };
+function transformRelatedExhibition(
+  exhibition: prismicT.RelationField<
+    'exhibitions',
+    'en-gb',
+    InferDataInterface<ExhibitionPrismicDocument>
+  >
```

Comment:
This is definitely at the more advanced end of our Prismic types; I only know how to do this by copy/pasting other examples.